website/docs/d/service_level_alert_helper.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_service_level_alert_helper"
+sidebar_current: "docs-newrelic-datasource-service-level-alert-helper"
+description: |-
+  Helper to set up alerts on Service Levels.
+---
+
+# Data Source: newrelic\_service\_level\_alert\_helper
+
+Use this data source to obtain the necessary fields to set up alerts on your service levels. It can be used for a `custom` alert_type in order to set up an alert with custom tolerated budget consumption and custom evaluation period or for recommended ones like `fast_burn`.
+
+## Example Usage
+
+Firstly set up your service level objective, we recommend to use local variables for the `target` and `time_window.rolling.count`, as they are also necessary for the helper.
+
+```hcl
+locals {
+  foo_target = 99.9
+  foo_period = 28
+}
+
+resource "newrelic_service_level" "foo" {
+    guid = "MXxBUE18QVBQTElDQVRJT058MQ"
+    name = "Latency"
+    description = "Proportion of requests that are served faster than a threshold."
+
+    events {
+        account_id = 12345678
+        valid_events {
+            from = "Transaction"
+            where = "appName = 'Example application' AND (transactionType='Web')"
+        }
+        good_events {
+            from = "Transaction"
+            where = "appName = 'Example application' AND (transactionType= 'Web') AND duration < 0.1"
+        }
+    }
+
+    objective {
+        target = local.foo_target
+        time_window {
+            rolling {
+                count = local.foo_period
+                unit = "DAY"
+            }
+        }
+    }
+}
+```
+Then use the helper to obtain the necessary fields to set up an alert on that Service Level.
+
+```hcl
+
+data "newrelic_service_level_alert_helper" "foo_custom" {
+    alert_type = "custom"
+    sli_guid = "MXxBUE18QVBQTElDQVRJT058MQ"
+    slo_target = local.foo_target
+    slo_period = local.foo_period
+    custom_tolerated_budget_consumption = 5
+    custom_evaluation_period = 90
+}
+
+resource "newrelic_nrql_alert_condition" "your_condition" {
+  account_id = 12345678
+  policy_id = 67890
+  type = "static"
+  name = "Successs (custom)"
+  enabled = true
+  violation_time_limit_seconds = 259200
+
+  nrql {
+    query = data.newrelic_service_level_alert_helper.foo_custom.nrql
+  }
+
+  critical {
+    operator = "above"
+    threshold = data.newrelic_service_level_alert_helper.foo_custom.threshold
+    threshold_duration = data.newrelic_service_level_alert_helper.foo_custom.evaluation_period
+    threshold_occurrences = "at_least_once"
+  }
+  fill_option = "none"
+  aggregation_window = 3600
+  aggregation_method = "event_flow"
+  aggregation_delay = 120
+  slide_by = 60
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+  * `alert_type` - (Required) The type of alert we want to set. Valid values are `custom` and `fast_burn`.
```

Comment:
Out of curiosity, why isn't `slow_burn` available?

Reply:
Additionally, I'd mention somewhere what `fast_burn` means in terms of thresholds and so on.

Reply:
slow_burn is not yet available due to some limitations, it will be available in the future